src/components/Scorecard.vue

```diff
@@ -350,7 +350,7 @@ export default defineComponent({
             transform: "scale(" + scale + ")",
             "transform-origin": "top center",
           },
-          filter: (node) => {
+          filter: (node: any) => {
```

Comment:
just don't assign `any` to every variable - find out what is the right type for them and use that. don't focus on just removing the error but on the right solution.

Reply:
I searched the error but no solution came up and also searched the type in ts still nothing, while doing the search I came across HTMLElement type but then got the same error.